lib/msal-common/src/telemetry/server/ServerTelemetryManager.ts

```diff
@@ -158,4 +165,70 @@ export class ServerTelemetryManager {
 
         return maxErrors;
     }
+
+    /**
+     * Get the region discovery fields
+     * 
+     * @returns string
+     */
+    getRegionDiscoveryFields(): string {
+        const regionDiscoveryFields: string[] = [];
+
+        regionDiscoveryFields.push(this.regionUsed ? this.regionUsed : "");
+        regionDiscoveryFields.push(this.getRegionSourceValue(this.regionSource));
+        regionDiscoveryFields.push(this.getRegionOutcomeValue(this.regionOutcome));
+
+        return regionDiscoveryFields.join(",");
+    }
+
+    /**
+     * Get the header value for the region source
+     * @param regionSource
+     * @returns string
+     */
+    getRegionSourceValue(regionSource: RegionDiscoverySources | undefined): string {
+        if (!regionSource) return "";
+
+        return regionSource;
+    }
+
+    /**
+     * Get the header value for the region outcome
+     * @param regionOutcome RegionDiscoveryOutcomes | undefined
+     * @returns string
+     */
+    getRegionOutcomeValue(regionOutcome: RegionDiscoveryOutcomes | undefined): string {
+        if (!regionOutcome) return "";
+
+        return regionOutcome;
+    }
+    
+    /**
+     * Get the header value for the cache information
+     * @returns string
+     */
+    getCacheInfoValue(): string {
+        if (this.forceRefresh) return "1";
```

Comment:
Isn't forceRefresh already accounted for when setting cacheOutcome? Do we need to duplicate this logic here?

Reply:
I have extended the cache outcome enum to include the forceRefresh option and removed this function.